### PR TITLE
Only read headers in draft mode to keep static optimization otherwise #539

### PR DESCRIPTION
**Pull request by pmi**
- also removed nagging logging
- fixed .gitignore

### CODE REVIEW

File: src/guillotine/fetchContent.ts [hunk 1]
```diff
@@ -33,6 +30,7 @@ import {combineMultipleQueries} from './combineMultipleQueries';
 import {fetchContentData} from './fetchContentData';
 import {applyProcessors} from './applyProcessors';
 import {createMetaData} from './createMetaData';
+import {draftMode, headers} from 'next/headers';
```
Comment (ComLock) [inline src/guillotine/fetchContent.ts:33]: Why did you move the import from next to the bottom? I typically try to order imports by distance and alphabetically:

type npmmodules
type /absolute
type ../upstream
type ./downstream
type ./same folder

npmmodules
/absolute
../upstream
./downstream
./same folder

**Review by ComLock — APPROVED** [review]

File: src/guillotine/fetchContent.ts (hunk 1 above)
Comment (pmi) [reply src/guillotine/fetchContent.ts:33]: It's not me, it's IntelliJ 👉

Comment (ComLock) [reply src/guillotine/fetchContent.ts:33]: Then I think your IntelliJ makes a lot of unnecessary changes.

Comment (pmi) [reply src/guillotine/fetchContent.ts:33]: Ok, I turned off import sorting in IntelliJ, but this file doesn't look particularly sorted to me anyway ;)

Comment (pmi) [reply src/guillotine/fetchContent.ts:33]: But in general, I don't know what logic one might use for sorting, so I think it's better to have some automatic sorting convention
